spec/ParseLiveQueryServer.spec.js

```diff
@@ -285,15 +285,15 @@ describe('ParseLiveQueryServer', function() {
       .catch(done.fail);
   });
 
-  it('can handle connect command', function() {
+  it('can handle connect command', async () => {
```

Comment:
Adding await here is fine. Can you add additional tests for this feature? You might have to await in other places.